Robot Code/src/org/usfirst/frc/team3316/robot/humanIO/SDB.java

```diff
@@ -156,6 +162,30 @@ private void initSDB ()
 		SmartDashboard.putData(new OpenIntake());
 		SmartDashboard.putData(new CloseIntake());
 		
+		// Climbing
+		SmartDashboard.putData(new ClimbingSequence());
+		SmartDashboard.putData(new OpenArmPiston());
+		SmartDashboard.putData(new CloseArmPiston());
+		SmartDashboard.putData(new MoveUp());
+		SmartDashboard.putData(new MoveDown());
+		
+		// Flywheel
+		SmartDashboard.putData(new JoystickFlywheel());
+		SmartDashboard.putData(new BangbangFlywheel());
+		SmartDashboard.putData(new PIDFlywheel());
+		
+		// Bangbang
+		putConfigVariableInSDB("flywheel_Bangbang_Setpoint");
```

Comment:
There's no need to still keep the flywheel variables here. We finished with them when we finished with the prototype.